sstables/sstables.cc

```diff
@@ -3020,6 +3006,15 @@ future<> sstable::wipe_storage(const sstring& name) noexcept {
         //    and notify the administrator about that for manual handling (rather than aborting).
         sstlog.warn("Failed to delete {}: {}. Ignoring.", name, std::current_exception());
     }
+
+    if (_storage.temp_dir) {
+        try {
+            co_await recursive_remove_directory(fs::path(*_storage.temp_dir));
+            _storage.temp_dir.reset();
```

Comment:
Hmm, I don't remember why this isn't done also on the error path...
No need to change it in this series in any case